pkg/serverinit/genconfig.go

```diff
@@ -733,14 +734,17 @@ func (b *lowBuilder) addGoogleDriveConfig(v string) error {
 	return nil
 }
 
-var errGCSUsage = errors.New(`genconfig: expected "googlecloudstorage" field to be of form "client_id:client_secret:refresh_token:bucket[/dir/]" or ":bucketname[/dir/]"`)
+var errGCSUsage = errors.New(`genconfig: expected "googlecloudstorage" field to be of form "client_id:client_secret:refresh_token:bucket[/dir/][:ratelimit]" or ":bucketname[/dir/]"`)
```

Comment:
It's not obvious what the format/unit of `ratelimit` is.

I think we should probably hold on making this configurable until the new configuration format is done. (I keep forgetting to finish it, but this is some extra motivation.)